Dockerfile

```diff
@@ -1,4 +1,15 @@
-FROM node:5.1.1
+FROM ubuntu
```

Comment:
@ahmedre As a developer, I am aware of how diff works. My comment was a general one and not specific to a line. It was a comment on the approach we are taking here. Building stable software means we build on top of a stable foundation.

Since the Node Foundation is providing us with an LTS version of the docker image it makes logical sense (to me) that a node app is built on top of the official image. Rather than building our own image. Because the Node foundation would have implemented many performance enhancement that many of us are not fully aware of, are not going to keep a top of, including security fixes.

For an open source project, it's also important that things are done to make the life of future developer easy because they would be the one who will keep the work going. If you build a custom image here, means they'll have to learn how to support/maintain the docker image as the app.